### ISSUE TITLE
Update compat for GeometryBasics v0.4

### ISSUE THREAD

**Issue by fjebaker** [JuliaGeometry/VoronoiCells.jl#30]
Hi, is it possible to have this package updated for GeometryBasics v0.4, and have the `compat` updated?

From what I can tell, `Rectangle` has been removed in the new minor version of GeometryBasics (as well as other changes), so the code in the README doesn't work (#26). Furthermore, adding compat `VoronoiCells = "0.2"` in a user's Project files forces GeometricBasics v0.3.13, which in turn may downgrade other packages a user is using (e.g. Makie.jl and more, which is causing me unresolvable issues).

Annoyingly, version v0.1.5 of VoronoiCells e1e0e280863b50fb0f5b0c127bc05ee08403622a didn't have a compat, so Pkg installs v0.1.5 by default when there are packages which require the latest version of GeometryBasics -- which is even more of a mess.

Thanks!

**Comment by robertdj** [comment]
Sure! Let me look into this next year :-) 

What version of Julia are you using?

**Comment by robertdj** [comment]
It's peculiar that two issues were raised recently about version 0.1.5. Is this the version Julia installed by default?
I don't think I can do anything about the missing `compat` section of version 0.1.5, but this should be fixed in the 0.2.x versions.

Regarding the `Rectangle`: This struct is from VoronoiCells and not GeometryBasics.

**Comment by fjebaker** [comment]
Hi @robertdj! Happy new year!

I attempted to recreate a small demonstration by installing GeometryBasics first and then VoronoiCells, but commits you've pushed in the last 24 hours successfully resolve the compatibility problem I was having:

```julia
(temp-test) pkg> status
      Status `~/Developer/temp-test/Project.toml`
  [5c1252a2] GeometryBasics v0.4.1
  [e3e34ffb] VoronoiCells v0.3.0
```

As for `Rectangle`, that was my oversight, I didn't think to check where the struct was being included from and made an assumption.

Thanks for fixing this so quickly! :)
